doc/man7/provider-base.pod

```diff
@@ -174,6 +187,74 @@ useless without at least provider_query_operation(), and
 provider_get_param_types() is fairly useless if not accompanied by
 provider_get_params().
 
+=head2 Core parameters
+
+core_get_params() understands the following known parameters:
+
+=over 4
+
+=item "openssl-version"
+
+This is a B<OSSL_PARAM_UTF8_PTR> type of parameter, pointing at the
+OpenSSL libraries' full version string, i.e. the string expanded from
+the macro B<OPENSSL_VERSION_STR>.
+
+=item "provider-name"
+
+This is a B<OSSL_PARAM_UTF8_PTR> type of parameter, pointing at the
+OpenSSL libraries' idea of what the calling provider is called.
+
+=back
+
+Additionally, any configuration parameter from the config file are
```

Comment:
Typo: parameter*s*

Reply:
I'm thinking that I should write "Additionally, provider specific configuration parameters from the config file are available, in dotted form."

If that sounds ok to you, that's what I'll change it to.

Reply:
Sounds fine to me

Reply:
Ok!